docs/device_registry_index.md

```diff
@@ -77,3 +77,6 @@ device_registry.async_get_or_create(
     sw_version=config.swversion,
 )
 ```
+
+### Battery
```

Comment:
Why is this a header 3 and be defined under defining devices? 

it also seems weird to put this in the device registry section since it has nothing to do with the registry ?

Reply:
This is the natural place for someone to look when setting up a device. This is also the only section that talks about the columns which appear in the UI for devices. I don't really care where the info goes but thought it might help someone who is setting up a device and not knowing why the battery isn't showing up even if they add the entity to a device.

Reply:
I agree that it helps, but we should not throw everything into the device registry introduction page. Lets add a new "Devices" section on the backend page. You can do that in `sidebars.json`.